programs/openbook-twap/src/lib.rs

```diff
@@ -376,7 +378,11 @@ pub mod openbook_twap {
         let market = ctx.accounts.market.load()?;
         let twap_market = &mut ctx.accounts.twap_market;
 
-        require!(market.time_expiry == 0, OpenBookTWAPError::NonZeroExpiry);
+        let current_time = Clock::get().unwrap().unix_timestamp as i64;
+        require!(
+            market.time_expiry > current_time + TEN_DAYS_IN_SECONDS,
+            OpenBookTWAPError::InsufficentExpiryTime
+        );
```

Comment:
nit: more natural IMO to cast the market.time_expiry to a `u64` than cast the clock to and have `TEN_DAYS_IN_SECONDS` be an `i64`

Reply:
I see what you mean. But I'm too busy, yolo

Reply:
Actually I did think about this. I think it's marginal. The difference is that we're casting / typing known values as i64, e.g. current time and ten days in seconds. But casting market.time_expiry "as u64" might do weird stuff. You have to have like a more formal way of doing it than just `as`.